lib/zip.js

```diff
@@ -33,22 +60,56 @@ function addVendorHelper() {
  */
 function removeVendorHelper() {
   if (this.options.zip && this.options.cleanupZipHelper) {
-    this.serverless.cli.log('Removing Python requirements helper...');
-    return fse.removeAsync(path.join(this.servicePath, 'unzip_requirements.py'));
+    if (this.serverless.service.package.individually) {
```

Comment:
This makes ziphelper setting stay global, which I think is ok, but a note for future devs: if you have a need for some functions to self-unzip, you will need to make this per-function.